docker/Dockerfile

```diff
@@ -30,6 +30,39 @@ COPY firefly/package.json firefly/yarn.lock ./
 RUN yarn install --ignore-platform --frozen-lockfile
 
 
+FROM node_module AS dev_env
+
+RUN apt-get update \
+# install tomcat9 for dev mode \
+    && curl -O https://dlcdn.apache.org/tomcat/tomcat-9/v9.0.64/bin/apache-tomcat-9.0.64.tar.gz \
```

Comment:
If you  are installing tomcat directly here. Should just do it in a base layer and not inherit `FROM tomcat:9.0-jdk17-openjdk-buster`

Reply:
We use a multi-stage Dockerfile.  This part of the build is in the `dev-env` stage.
Docker is smart enough to know that if you request to build the `final` stage, it will traverse the stage's dependencies and skip this `dev-env` stage.  Therefore, this tomcat9 install line will not be executed during a normal firefly build, only for DevMode.